src/components/Popup/PopupContent.vue

```diff
@@ -16,9 +16,8 @@
         :height="height"
```

Comment:
Joe also mentioned that clicking outside of the modal should NOT close the popup, which is out of scope for this PR, but I believe could be accomplished by removing line 7 if you want to give that a try